scipy/io/wavfile.py

```diff
@@ -489,54 +519,124 @@ def _handle_pad_byte(fid, size):
         fid.seek(1, 1)
 
 
-def read(filename, mmap=False):
+def read(filename, mmap=False, int_form='lj'):
     """
     Open a WAV file
 
-    Return the sample rate (in samples/sec) and data from a WAV file.
+    Return the sample rate (in samples/sec) and data from an LPCM WAV file.
 
     Parameters
     ----------
     filename : string or open file handle
-        Input wav file.
+        Input WAV file.
     mmap : bool, optional
-        Whether to read data as memory-mapped.
-        Only to be used on real files (Default: False).
+        Whether to read data as memory-mapped (default: False).  Not compatible
+        with some bit depths; see Notes.  Only to be used on real files.
 
         .. versionadded:: 0.12.0
+    int_form : {'lj', 'rj', 'fp', 'fs'}
+        The format ['int_format', 'type', 'dtype', 'format_'?] of the returned
```

Comment:
Otherwise this `format_` creates restructuredtext problems (see warnings in CircleCI build logs)
```suggestion
        The format ``['int_format', 'type', 'dtype', 'format_'?]`` of the returned
```

Reply:
these are just suggestions of other possible parameter names that would be removed after we decide on what it should be called

Reply:
like should the parameter be written `format='native'` or `int_format='lj'` or `dtype='native'` etc.